src/models/image-map.ts

```diff
@@ -38,9 +37,9 @@ export interface IImageContext {
   key?: string;
 }
 export interface IImageBaseOptions {
-  curriculumBaseUrl?: string;
   filename?: string;
   unitCodeMap?: Record<string, string>;
+  uniturl?: string;
```

Comment:
Are these two properties used? 
The `uniturl` has the wrong capitalization. 
And I looked through the code and I don't think they are ever passed in by callers of getImage
And they don't seem to be used by any of the `store(..)` implementation. 
I think only the localAsset handler uses their values and it gets them out of the imageMap instead from the options.

Based on all of that I'd guess you could just remove these two properties and everything should continue to work the same way.

Reply:
Yeah, they don't seem to be used. Removing them didn't cause any problems. I must've gotten a little mixed up trying to get the typing right.